aws/table_aws_ec2_settings.go

```diff
@@ -0,0 +1,137 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEc2Settings(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ec2_settings",
+		Description: "AWS EC2 Settings",
+		List: &plugin.ListConfig{
+			Hydrate: listAllAwsRegions,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "default_ebs_encryption_enabled",
+				Description: "Indicates whether encryption by default is enabled.",
+				Type:        proto.ColumnType_BOOL,
+				Hydrate:     getDefaultEBSVolumeEncryption,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "default_ebs_encryption_key",
+				Description: "The Amazon Resource Name (ARN) or alias of the default CMK for encryption by default.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getDefaultEBSVolumeEncryptionKey,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromConstant("EC2 Settings"),
```

Comment:
To better differentiate which row is from which region, are we able to change this to `<region> EC2 Settings`?